### DISCUSSION TITLE
Steam 已经很久没有更新了，还有更新steam版本吗？

### DISCUSSION THREAD

**Discussion by Remonli** [jynew/jynew#1175] [General]
是不是以后要更新都要自己打包了。

**Comment by MephistoSatanDevil** [comment]
> 是不是以后要更新都要自己打包了。

已经正式上架了，地址：https://store.steampowered.com/app/2098790/

**Reply by Remonli** [reply]
这个连接是一个mod哇，渡城残魂传？

**Reply by MephistoSatanDevil** [reply]
> 这个连接是一个mod哇，渡城残魂传？

只是平台附带的一个示例游戏。金庸群侠传3D重制版有人放在创意工坊里了。

**Comment by ghost** [comment]
iOS版的mod在哪里可以下载到？

**Reply by chenggongchina** [reply]
目前需要民间自己搭建HTTP服务器，因为很多MOD是版权有争议的，你懂的。
